internal/tasks/process-file-updates-task/process-file-updates-task.yaml

```diff
@@ -236,14 +238,75 @@ spec:
             error="\"no replacements were performed\"" \
             yq -o json --null-input '.str = strenv(error), .error = strenv(error)' \
             | tee "$(results.fileUpdatesInfo.path)"
-            echo -n "Failed" |tee "$(results.fileUpdatesState.path)"
+            if [[ "$keyNotFound" == true ]]; then
+                echo -n "Failed" |tee "$(results.fileUpdatesState.path)"
+            else
+                echo -n "Success" |tee "$(results.fileUpdatesState.path)"
+            fi
             # it should exit 0 otherwise the task does not set the results
             # this way the InternalRequest can see what was wrong
             exit 0
         fi
 
         echo -e "\n*** START LOCAL CHANGES ***\n"
-        git diff
+        git diff | tee "${TEMP}"/tempMRFile.diff
+
+        if [[ -s "${TEMP}"/tempMRFile.diff ]]; then
+            # replacements exist
+            # It's to deal with the lines like "@@ -N1,N2 +N3,N4 @@", 
+            # but it's possible to meet a line like "@@ -4,7 +4,7 @@ $schema: /app/app-settings.yml".
+            awk '/^@@/ {match($0, /@@ ([^@]+) @@/, arr); print arr[1]}' "${TEMP}"/tempMRFile.diff |\
+               tee "${TEMP}"/lineFile
+        fi
+
+        openMRList=$(glab mr list -R "${UPSTREAM_REPO}" --search "Konflux release" |grep "^!"| cut -f1 | tr -d '!')
+        for mrNum in ${openMRList} ; do
+            foundMR=false
+            glab mr diff "${mrNum}" --repo "${UPSTREAM_REPO}" > "${TEMP}"/oneMR.diff 
+
+            if [[ ! -s "${TEMP}"/lineFile ]]; then
+                # only seed files  
+                foundMR=false
```

Comment:
Is this needed since it is set on line 264?

Reply:
Removed. Thanks